### ISSUE TITLE
When are you going to give us a new official release?

### ISSUE THREAD

**Issue by Lewdtim** [arichornlover/uYouEnhanced#794]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

✅ Yes, I believe this is a bug. I will open a new issue with the bug template

### My question

When are you going to give us a new official release?

The last one was a long time ago.
I don’t want to create a tweak ipa with an official release.

please let us know as your github is getting updated regularly, after its comeback.

also thanks for all of this

**Comment by yodaluca23** [comment]
https://github.com/arichornlover/uYouEnhanced/wiki/Building#respect-intellectual-property---important

**Comment by Lewdtim** [comment]
> You have to build it yourself, it's honestly not that hard

I know how to build it, but if he just officially release one !

**Comment by PlasticIngot** [comment]
> > ### My question
> > When are you going to give us a new official release?
> > The last one was a long time ago. I don’t want to create a tweak ipa with an official release.
> > please let us know as your github is getting updated regularly, after its comeback.
> > also thanks for all of this
> 
> Unfortunately this repository received this dmca from google earlier this year. https://github.com/github/dmca/blob/master/2024/03/2024-03-06-google.md

yeah but just releasing the tweak build? without ipa wont work ?

**Comment by sorae42** [comment]
> > > ### My question
> > > When are you going to give us a new official release?
> > > The last one was a long time ago. I don’t want to create a tweak ipa with an official release.
> > > please let us know as your github is getting updated regularly, after its comeback.
> > > also thanks for all of this
> > 
> > 
> > Unfortunately this repository received this dmca from google earlier this year. https://github.com/github/dmca/blob/master/2024/03/2024-03-06-google.md
> 
> yeah but just releasing the tweak build? without ipa wont work ?

This thing requires the original YT app to work, it literally injecting the tweaks into the app.

**Comment by yodaluca23** [comment]
> This thing requires the original YT app to work, it literally injecting the tweaks into the app.

Yes, but that doesn't mean you have to distribute it with the YouTube app. How do you think any other jailbreak tweak is normally made it could be compiled into a .deb I don't know how hard it would be. I know uYou+ had like I think one release with a .deb release someone could probably make a workflow for it...

**Comment by sorae42** [comment]
> > This thing requires the original YT app to work, it literally injecting the tweaks into the app.
> 
> Yes, but that doesn't mean you have to distribute it with the YouTube app. How do you think any other jailbreak tweak is normally made it could be compiled into a .deb I don't know how hard it would be. I know uYou+ had like I think one release with a .deb release someone could probably make a workflow for it...

It's possible, but uYouEnhanced are made to mainly for non-jailbroken users. If you are jailbroken then why not installing the tweaks directly?

**Comment by yodaluca23** [comment]
> > > This thing requires the original YT app to work, it literally injecting the tweaks into the app.
> 
> > 
> 
> > Yes, but that doesn't mean you have to distribute it with the YouTube app. How do you think any other jailbreak tweak is normally made it could be compiled into a .deb I don't know how hard it would be. I know uYou+ had like I think one release with a .deb release someone could probably make a workflow for it...
> 
> 
> 
> It's possible, but uYouEnhanced are made to mainly for non-jailbroken users. If you are jailbroken then why not installing the tweaks directly?

Because then there could be distributed releases as OP wants..?
